packages/modules/notification/src/migrations/Migration20240509083918_InitialSetupMigration.ts

```diff
@@ -0,0 +1,29 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240509083918_InitialSetupMigration extends Migration {
+  async up(): Promise<void> {
+    this.addSql("drop table if exists notification")
+    this.addSql("drop table if exists notification_provider")
```

Comment:
q: are we loosing any data from v1 if someone runs it in the same database? I suppose this has been validated elsewhere but just raising the question

Reply:
@adrien2p our current setup doesn't support any migrations from v1 so I added this as a quick way to have v2 running (as we have other places where we eg. drop columns). I think we'll review all migrations before anyone attempts any sort of migration 😄